MVC/app.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express')
+const app = express()
+// my different cb functions form controllers
+const getTopics = require('./controllers')
+
+app.use(express.json())
```

Comment:
we dont need this line yet - it will be needed when we start making post requests. Try to make sure only the relevant code is included in each branch